debian/control

```diff
@@ -21,6 +21,6 @@ Vcs-Git: https://github.com/hwangsaeul/gaeul.git
 
 Package: gaeul
 Architecture: any
-Depends: libgaeguli1, libchamge1, ${shlibs:Depends}, ${misc:Depends}
+Depends: adduser, ${shlibs:Depends}, ${misc:Depends}
```

Comment:
Just curious about why dependencies `libgaeguli1` and `libchamge1` are removed.

Reply:
`dpkg-buildpackage` should figure out shared library dependencies on its own by inspecting the created ELF binaries and substitute `${shlibs:Depends}` with an auto-generated list of corresponding packages. Maintaining `Depends:` field manually is thus unnecessary work (when it comes to shared libraries; adduser is a different beast!).

See [man dpkg-shlibdeps](https://manpages.debian.org/unstable/dpkg-dev/dpkg-shlibdeps.1.en.html) for details.